app/player/player.routes.js

```diff
@@ -9,4 +9,7 @@ module.exports = (app, db, middlewares, Sentry) => {
     app.route('/player/returning').get(middlewares, controller.getReturningProduction);
     app.route('/stats/player/season').get(middlewares, controller.getSeasonStats);
     app.route('/player/portal').get(middlewares, controller.getTransferPortal);
-};
\ No newline at end of file
+    app.route('/player/active').get(middlewares, controller.getActivePlayers);
```

Comment:
The more I think about it, I think these three endpoints should be consolidated into a single endpoint with optional query parameters for `position`, 'division` (fbs,fcs), and `conference`. Maybe a boolean flag for `powerFiveOnly`. I'm kind of iffy on having params for "power 5" since that is a very fluid concept as we've gone from BCS to P5 to whatever the heck we have starting in 2024 when the concept of P5 will cease to exist.